utils/send_email.py

```diff
@@ -0,0 +1,13 @@
+from django.conf import settings
+from django.core.mail import send_mail
+
+
+def send_email_notification(subject, message, recipient_list):
```

Comment:
This is blocking the server instance until it finishes, you should do it asynchronously!